niworkflows/interfaces/surf.py

```diff
@@ -558,6 +558,61 @@ def _run_interface(self, runtime):
         return runtime
 
 
+class CreateSurfaceROIInputSpec(TraitedSpec):
+    subject_id = traits.Str(desc='subject ID')
+    hemisphere = traits.Enum(
+        "L",
+        "R",
+        mandatory=True,
+        desc='hemisphere',
+    )
+    thickness_file = File(exists=True, mandatory=True, desc='input GIFTI file')
+
+
+class CreateSurfaceROIOutputSpec(TraitedSpec):
+    roi_file = File(desc='output GIFTI file')
+
+
+class CreateSurfaceROI(SimpleInterface):
+    """Prepare GIFTI shape file for use in"""
```

Comment:
Looks like I failed to comment this one properly.

```suggestion
    """Prepare GIFTI shape file for use in cortical masking

    Distilled from the FreeSurfer2CaretConvertAndRegisterNonlinear.sh script in
    DCAN-HCP PostFreeSurfer scripts (as of commit 9291324). The relevant lines
    are 277-290.
    """
```